pkg/console/subresource/deployment/deployment.go

```diff
@@ -92,110 +88,50 @@ func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap,
 	if canMountCustomLogo {
 		volumeConfig = append(volumeConfig, customLogoVolume())
 	}
-
+	deployment.Spec.Template.Spec.Containers = []corev1.Container{
+		consoleContainer(operatorConfig, volumeConfig, proxyConfig),
+	}
 	podAnnotations := map[string]string{
 		workloadManagementAnnotation: workloadManagementAnnotationValue,
```

Comment:
I just realized workloadManagementAnnotationValue is a fixed value in both manifests... I'll move this tomorrow
```
workloadManagementAnnotationValue                    = `{"effect": "PreferredDuringScheduling"}`
```